public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, colorManipulator } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+export default function PubdashFooter() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.footer}>
+      <span className={styles.logoText}>
+        <a href="https://grafana.com/" target="blank">
+          powered by Grafana <img className={styles.logoImg} src="public/img/grafana_icon.svg"></img>
+        </a>
+      </span>
+    </div>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  footer: css`
+    position: absolute;
+    height: 2rem;
```

Comment:
Could we use `px` here instead?

Reply:
When should we be using `px` instead of our theme spacing?

Reply:
Spacing is mostly for margins/paddings, for hight/width we usually use `px`, but it's quite seldom that we need to specify exact height for the elements.

Reply:
will change back to px